Dockerfile

```diff
@@ -4,3 +4,7 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 
 COPY *.py /usr/local/bin/
 
+WORKDIR /
+
+COPY requirements.txt /
+RUN pip3 install --no-cache-dir -r requirements.txt
```

Comment:
Also because you're in a fresh container, you should probably `pip3 install --upgrade pip && pip3 install...`